app/views/groups/self_registration/_aside_sektion.html.haml

```diff
@@ -2,7 +2,7 @@
 -#  Affero General Public License version 3 or later. See the COPYING file at the top-level directory
 -#  or at https://github.com/hitobito/hitobito.
 
-= turbo_frame_tag(:aside, class: '.col-md') do
+= turbo_frame_tag(partial_name, class: 'col-md') do
   - if wizard.is_a?(Wizards::Signup::SektionWizard) && !wizard.last_step?
```

Comment:
The check here shouldn't be necessary anymore, or is it still used in multiple places?